internal/pkg/cli/pipeline_init.go

```diff
@@ -261,12 +264,12 @@ func (opts *InitPipelineOpts) createIntegTestBuildspecPerApp(apps []archer.Manif
 
 	paths := make([]string, 0, len(apps))
 	for _, app := range apps {
-		path, err := opts.workspace.WriteFile([]byte(content), app.IntegTestBuildspecPath())
+		err := opts.fsUtils.WriteFile(app.IntegTestBuildspecPath(), []byte(content), 0644)
```

Comment:
Should file writes be a responsibility of the `workspace` to ensure they're relative to the workspace?